php/Dockerfile

```diff
@@ -88,14 +81,16 @@ ENTRYPOINT ["docker-custom-entrypoint"]
 
 # ========================================
 FROM proximis_php as proximis_php_blackfire
+
```

Comment:
You could take [this image](https://github.com/origamiphp/docker-images/blob/main/php/7.4/blackfire/Dockerfile) as an example to simplify this instruction. 😉